cclib/parser/molcasparser.py

```diff
@@ -222,38 +222,41 @@ def extract(self, inputfile, line):
         #  Threshold at which QNR/C2DIIS is turned on 0.75E-01
         #  Threshold for Norm(delta) (QNR/C2DIIS)     0.20E-04
         if line[:34] == '++    Optimization specifications:':
-
-            scftargets = []
-            while not line[6:37] == 'Threshold for SCF energy change':
+            self.skip_lines(inputfile, ['d', 'b'])
+            line = next(inputfile)
+            if line.strip().startswith('SCF'):
+                scftargets = []
+                self.skip_lines(inputfile, ['Minimized', 'Number', \
+                                         'Maximum', 'Maximum'])
```

Comment:
I think that if you start the list on the next line you won't need to use the backslash.

Reply:
Yes, agreed!